styles.css

```diff
@@ -133,3 +135,937 @@ li {
+}
+
+.Img-Placeholder {
+  position: static;
+  width: 327px;
+  height: 237px;
+  left: 24px;
+  top: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.title-post {
+  position: static;
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 0;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.supporting-text {
+  position: static;
+  width: 327px;
+  height: 147px;
+  left: 0;
+  top: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  position: static;
+  width: 297px;
+  height: 40px;
+  left: 0;
+  top: 207px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 12px -17px;
+}
+
+.tag-1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 51px;
+  height: 40px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-right: -8px;
+}
+
+.tag-2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 61px;
+  height: 40px;
+  left: 61px;
+  top: 0;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-right: -8px;
+}
+
+.tag-3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 106px;
+  height: 40px;
+  left: 129px;
+  top: 0;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-right: -8px;
+}
+
+.tag-4 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 63px;
+  height: 40px;
+  left: 236px;
+  top: 0;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin-right: -8px;
+}
+
+button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: static;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  top: 259px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.see-project {
+  position: static;
+  width: 102px;
+  height: 24px;
+  left: 12px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
```

Comment:
Also the text of your button projects should be center, please try to do that 

- [ ] Kindly remove the border from here also 

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164911976-b0ff2895-c348-4ec9-a2fb-5e165bf668ab.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164911995-7ed45222-bd43-43e6-9e35-16e5a636b20b.png)

Reply:
done